src/components/screen/Chat.tsx

```diff
@@ -95,6 +97,35 @@ function Screen(props: Props): React.ReactElement {
     setIsSending(true);
   };
 
+  const getPermissions = async (
```

Comment:
Looks like you are using the `same` function and codes are just `cp` from elsewhere. It would be better to be placed globally somewhere like in `utils`.